examples/rgbpp/shared/utils.ts

```diff
@@ -0,0 +1,35 @@
+import * as fs from 'fs';
+import * as path from 'path';
+
+/**
+ * Save ckbVirtualTxResult to a log file
+ * @param ckbVirtualTxResult - The ckbVirtualTxResult to save
+ * @param exampleName - Example name used to distinguish different log files
+ */
+export const saveCkbVirtualTxResult = (ckbVirtualTxResult: unknown, exampleName: string) => {
```

Comment:
You can define a new type for the `ckbVirtualTxResult` in the `utils` file, for example:
```
export type CkbVirtualTxResultType = BaseCkbVirtualTxResult | SporeVirtualTxResult | SporeCreateVirtualTxResult |  SporeTransferVirtualTxResult
```
Use the `CkbVirtualTxResultType` to replace the `unknown`